rover_controller:
  ros__parameters:
    max_linear_velocity: 0.5
    max_angular_velocity: 1.0
    goal_tolerance: 0.2
    planning_frequency: 2.0

obstacle_detector:
  ros__parameters:
    obstacle_distance_threshold: 2.0
    min_obstacle_points: 5

vision_processor:
  ros__parameters:
    detection_threshold: 0.5
    image_processing_rate: 10.0

flask_bridge:
  ros__parameters:
    flask_host: "0.0.0.0"
    flask_port: 5000